### ISSUE TITLE
Shortcuts Integration

### ISSUE THREAD

**Issue by Rishi5813** [thebaselab/codeapp#38]
It would be great if Code App could integrate with Shortcuts. For Example if we would like to run or open as a workspace folder in This App.

**Comment by wisinfun** [comment]
I also have similar requirements. I hope to execute a node script or a node file in shortcuts @bummoblizard

**Comment by Lukas-Batema** [comment]
I agree, please somehow add it to the toolbar as Replit mobile does (if needed, I can provide screenshots), and I may start using this app again.
